library/pod-security-policy/users/template.yaml

```diff
@@ -15,60 +18,100 @@ spec:
           properties:
             runAsUser:
               type: object
+              description: "Controls which user ID values are allowed in a Pod or container level SecurityContext."
               properties:
                 rule:
                   type: string
+                  description: "A strategy for applying the runAsUser restriction."
+                  enum:
+                    - MustRunAs
+                    - MustRunAsNonRoot
+                    - RunAsAny
                 ranges:
                   type: array
+                  description: "A list of user ID ranges affected by the rule."
                   items:
                     type: object
+                    description: "The range of user IDs affected by the rule."
                     properties:
                       min:
                         type: integer
+                        description: "The minimum user ID in the range, inclusive."
                       max:
                         type: integer
+                        description: "The maximum user ID in the range, inclusive."
             runAsGroup:
               type: object
+              description: "Controls which group ID values are allowed in a Pod or container level SecurityContext."
               properties:
                 rule:
                   type: string
+                  description: "A strategy for applying the runAsGroup restriction."
+                  enum:
+                    - MustRunAs
+                    - MayRunAs
+                    - RunAsAny
                 ranges:
                   type: array
+                  description: "A list of group ID ranges affected by the rule."
                   items:
                     type: object
+                    description: "The range of group IDs affected by the rule."
                     properties:
                       min:
                         type: integer
+                        description: "The minimum group ID in the range, inclusive."
                       max:
                         type: integer
+                        description: "The maximum group ID in the range, inclusive."
             supplementalGroups:
               type: object
+              description: "Controls the supplementalGroups values are allowed in a Pod or container level SecurityContext."
```

Comment:
"Controls the supplementalGroups values that are allowed in a Pod or container-level SecurityContext."